numpy/linalg/tests/test_linalg.py

```diff
@@ -1039,6 +1039,18 @@ def tz(mat):
             if dt != object:
                 tz(self.stacked.astype(dt))
 
+    def test_power_is_three(self, dt):
```

Comment:
The case of the exponent being 3 (a hard-coded case) was actually untested before.